README.md

```diff
@@ -273,10 +273,12 @@ node verifyBucketSproxydKeys.js
 * One of:
 
   * **BUCKETS**: comma-separated list of buckets to scan
+  * Example : `BUCKETS=bucket1,bucket2,bucket3`
 
 * or:
 
   * **RAFT_SESSIONS**: comma-separated list of raft sessions to scan
+  * Example : `docker run --net=host -e 'BUCKETD_HOSTPORT=127.0.0.1:9000' -e 'SPROXYD_HOSTPORT=127.0.0.1:8181' -e 'RAFT_SESSIONS=1' registry.scality.com/s3utils/s3utils:1.12.5 node verifyBucketSproxydKeys.js`
```

Comment:
For long commands I suggest splitting it up on different lines, something like:
```suggestion
  * Example : ```docker run \
   --net=host \
   -e 'BUCKETD_HOSTPORT=127.0.0.1:9000' \
   -e 'SPROXYD_HOSTPORT=127.0.0.1:8181' \
   -e 'RAFT_SESSIONS=1' \
   registry.scality.com/s3utils/s3utils:1.12.5 \
   node verifyBucketSproxydKeys.js```
```
Just an idea though, same applies to your other example